src/routes/tokenGovernance.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { Tabs } from 'antd';
+
+import Web3SubmitWarning from '../components/web3SubmitWarning';
+
+import { Button, Form, Input, Row, Col } from 'antd';
+import TextArea from 'antd/lib/input/TextArea';
+import { keccak256 } from 'ethereumjs-util';
+import Web3 from '../stores/web3/ts_workaround';
+import { tokenGovernanceContractStore } from '../stores/tokenGovernanceContract';
+
+const { Fragment } = React;
+const { TabPane } = Tabs;
+
+export default class TokenGovernance extends React.Component {
+  public state: { title: string; description: string };
+  public sendProposal: any;
```

Comment:
I would recommend to use typescript (here and everywhere else) as it helps to catch errors earlier and make it easier to reason about the code. E.g. define a proper type for `sendProposal` here